src/WebJobs.Script.WebHost/Models/HostAssignmentZipUrl.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    public class HostAssignmentZipUrl
+    {
+        public HostAssignmentZipUrl(string envVar, string url)
```

Comment:
seems this can be RunFromPackageContext. what do you think?

Reply:
Sounds better since it contains more than just the url.